src/large_pages/node_large_page.cc

```diff
@@ -294,6 +296,25 @@ static bool IsSuperPagesEnabled() {
 }
 #endif
 
+int IsLargePagesEnabled() {
```

Comment:
Can you name this e.g. `GetLargePagesSupport()`? `IsLargePagesEnabled()` suggests it returns a boolean when it doesn't.

Alternatively, you could just inline it at the call site; there's only one caller.